src/uct/ib/rc/base/rc_iface.h

```diff
@@ -619,4 +628,10 @@ uct_rc_iface_send_op_set_name(uct_rc_iface_send_op_t *op, const char *name)
 #endif
 }
 
+static UCS_F_ALWAYS_INLINE int
+uct_rc_iface_is_flush_remote(uct_rc_iface_t *iface)
```

Comment:
do we really need wrapper for this check?

Reply:
unresolved

Reply:
yep, sorry.
removed